### DISCUSSION TITLE
executing external native build for ndkBuild Affected Modules: core

### DISCUSSION THREAD

**Discussion by jhwsx** [shadowsocks/shadowsocks-android#2687] [Q&A]
**Describe the bug**
executing external native build for ndkBuild /home/wzc/op/shadowsocks-android/core/src/main/jni/Android.mk
Affected Modules: core

**To Reproduce**
When I compile the project, it can always reproduce.

**Expected behavior**
Compile pass.

**Screenshots**
![xxxxx](https://user-images.githubusercontent.com/19379538/77624677-258fd600-6f7d-11ea-9a60-863a606ba0b8.png)

**Smartphone (please complete the following information):**
No device, I cannot compile it ok, let alone install it.

**Configuration**
I don't know about the configuration.

**Additional context**
OS system: Ubuntu 18.0.4
AndroidStudio version: 4.0 beta 3
gradle: 4.0.0-beta03 and 6.1.1

In fact, I just clone the master branch and work on it.

The only place I modified is：
change this line:
```
classpath 'com.android.tools.build:gradle:4.0.0-alpha09'
```
to this line:
```
classpath 'com.android.tools.build:gradle:4.0.0-beta03'
```
I wonder whether this modification really matters.
And I want to compile pass.

**Comment by kusty456** [comment]
> **Describe the bug**
,>
> 
> 
> **To Reproduce**
> When I compile the project, it can always reproduce.
> 
> 
> Compile pass.
> 
> **Screenshots**
> !](https://user-images.github.com/
> 
> **Smartphone 
> .
> 
> **Configuration**
> 
> 
> **Additional 
> system: Ubuntu
> AndroidStudio 
> gradle:
> 
> In fact, I the master branch and work on it.
> 
> The only place I modified is：
> change this line:
> ```
> 'com.android.tools'
> ```
> to this line:
> ```
>
>
> 
>

**Comment by jhwsx** [comment]
Why is this an invalid question?
If you need more information, I can give more details; If this is a problem which has had an answer, please reference it.
I'm not here to issue an invalid question, I am looking for an answer which makes me struggling for days.

**Comment by ghost** [comment]
I am having the same problem, I followed the actions in the documentation, why should I close this issue? If there is an answer please send me a link
